src/components/app/ProfileFilterNavigator.js

```diff
@@ -2,6 +2,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+// @flow
+
```

Comment:
Done, but I'm not sure what else I can do on this component.  If there is more to be done please point me in the right direction.  Thanks 😄